File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/M. Capital or Small or Digit/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/M

#include<iostream>
using namespace std;
 
int main(){
    char c;
    cin >> c;
    if(c >= 48 && c <= 57){
        cout << "IS DIGIT" << endl;
        return 0;
    } else{
        cout << "ALPHA" << endl;
    }
    if (c >= 'A' && c <= 'Z'){
        cout << "IS CAPITAL" << endl;
    } else{
        cout << "IS SMALL" << endl;
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/M. Capital or Small or Digit/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/K

using System;

namespace Codeforces.Assuit.Sheet1
{
    class ProblemK
    {
        static void Main(string[] args)
        {
            char c = Convert.ToChar(Console.ReadLine()); // take one line input
            if (c >= 48 && c <= 57)
            {
                Console.WriteLine("IS DIGIT");
                return;
            }
            else
            {
                Console.WriteLine("ALPHA");
            }
            if (c >= 'A' && c <= 'Z')
            {
                Console.WriteLine("IS CAPITAL");
            }
            else
            {
                Console.WriteLine("IS SMALL");
            }

        }
    }
}
